modules/installation-configuration-parameters.adoc

```diff
@@ -1159,6 +1203,31 @@ The GCP Compute Engine System service account email, like `<service_account_name
 |For compute machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
+|`compute.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP service account used for the encryption request for compute machines. If absent, the Compute Engine default service account is used. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
+|The GCP service account email, for example `<service_account_name>@<project_id>.iam.gserviceaccount.com`.
+
+|`compute.platform.gcp.osDisk.diskSizeGB`
+|The size of the disk in gigabytes (GB). This value applies to compute machines.
+|Any size between 16 GB and 65536 GB.
+
+|`compute.platform.gcp.osDisk.diskType``
```

Comment:
One additional back-quote in the end, please remove.

Reply:
Besides, my screen which is 1920 x 1080 resolution cannot show all the 3 columns of the table, if without horizontal-scrolling. To be honest, I have to open 2 windows, one to show the first column of the table, another to show the other 2 columns. Is it possible to enhance? Thanks!

Reply:
Thanks, I fixed the typo. I don't think there is anything we can do to help with the table formatting. It might be easier to just view the source code rather than the rendered output. Sorry!